.github/workflows/build-ffmpeg.yml

```diff
@@ -90,11 +90,10 @@ jobs:
               --enable-videotoolbox
           - name: Darwin-arm64
             os_type: macos
-            os: macos-12
+            os: macos-14
```

Comment:
Do you know if this build will work on arm64 if macOS is < 14?

Reply:
Unfortunately, I don't have a Apple Silicon Mac still running Sonoma to test with. It looks like it should since FFmpeg uses `__builtin_available()` to check for feature availability (and none of the `__builtin_available()` checks are looking for features in anything newer than macOS 13 anyway).

Reply:
Okay cool. This looks fine to merge.